cartographer/cloud/internal/map_builder_server.h

```diff
@@ -110,8 +110,8 @@ class MapBuilderServer : public MapBuilderServerInterface {
   void ProcessSensorDataQueue();
   void StartSlamThread();
   void OnLocalSlamResult(
-      int trajectory_id, common::Time time, transform::Rigid3d local_pose,
-      sensor::RangeData range_data,
+      int trajectory_id, const std::string& client_id, common::Time time,
```

Comment:
the other arguments are passed by value, also pass client_id by value?

Reply:
done.